.github/pull_request_template.md

```diff
@@ -6,12 +6,9 @@ Describe the content of the PR and links to related issues, bugs of features.
 
 Before submitting this PR, please make sure:
 
-- [ ] The code builds cleanly without new errors or warnings
-- [ ] The code passes all the existing tests
-- [ ] If this PR adds a new feature, new tests have been added to test these
-new features
-- [ ] All relevant documentation has been updated or added
-
+- [ ] You have run the code checks, tests and documentation build successfully
```

Comment:
For me, these checkboxes are just noise, but I can see that others have a different opinion.

Reply:
To be honest, I thought the checkboxes were a good idea at the beginning, but it turns out I never use them. I now think it would be better to simplify the PR template with a couple of sentences reminding the user to add a good description of the changes made on the PR, run the  tests and check the coding guidelines/review checklist before submitting

Reply:
What do you think of a template in following or related form, 
where the goal is also to help structure the description?

## Description
fixes #

### Relevant Tests

### Added

### Changed

### Removed

## Please Note

* Do not expect us to review PRs with failing checks. You may, however ask for advice about failing checks.
* The more complete your description the quicker the review.
* The coding guidelines by which we assess PRs are here (link).
* New authors must sign a legal ... etc

Reply:
Obviously the above is only a rough example